### PR TITLE
patch for issue #2652 now runs on python 3.x

### PR DESCRIPTION
**Pull request by fenderrex**

### CODE REVIEW

**Comment by fenderrex** [comment]
Hey google bot I signed it!

**Comment by fenderrex** [comment]
not sure if a new global was a good idea in this case, thoughts?

File: research/object_detection/builders/model_builder_test.py [hunk 1]
```diff
@@ -40,7 +40,11 @@
     frcnn_resnet_v1.FasterRCNNResnet152FeatureExtractor
 }
 
-
+if sys.version_info[0] == (3):
+	FEATURE_EXTRACTOR_MAPS_ITEMS= FEATURE_EXTRACTOR_MAPS.items()
```
Comment (tombstone) [inline research/object_detection/builders/model_builder_test.py]: Can you revert this file. I submitted a patch for this already.

**Comment by fenderrex** [comment]
I apologize if I'm making a mess with git I don't have a whole lot of experience I suppose I should make my own branch for development on my fork as everything I push ends up as part of this pull request for the upstream repo? Could I have just made a pull request from the commit its self and not the head pointer? is there a standard practice to how Google uses git?

**Comment by fenderrex** [comment]
@tensorflow-jenkins hello?

**Pull request closed without merging**